packages/rolldown/src/plugin/index.ts

```diff
@@ -80,7 +80,7 @@ export interface SourceDescription extends Partial<PartialNull<ModuleOptions>> {
 export interface ResolveIdExtraOptions {
   custom?: CustomPluginOptions;
   isEntry: boolean;
-  kind: 'import' | 'dynamic-import' | 'require-call';
+  kind: BindingHookResolveIdExtraArgs['kind'];
```

Comment:
The rollup api is using `import`, change it to 'import-statement' will caused the break changes.